tests/test_validators.py

```diff
@@ -151,6 +152,32 @@ def test_validate_prefix_overlap():
     with pytest.raises(ValidationError) as exc:
         validate_prefix_overlap("198.32.124.0/23")
 
+@pytest.mark.parametrize("value,validated", [
+    # success validation
+    ("RIPE::AS-FOO", "RIPE::AS-FOO"),
+    ("AS-FOO@RIPE", "AS-FOO@RIPE"),
+    ("ripe::as-foo", "RIPE::AS-FOO"),
+    ("as-foo@ripe", "AS-FOO@RIPE"),
+    ("as-foo@ripe as-bar@ripe", "AS-FOO@RIPE AS-BAR@RIPE"),
+    ("as-foo@ripe,as-bar@ripe", "AS-FOO@RIPE AS-BAR@RIPE"),
+    ("as-foo@ripe, as-bar@ripe", "AS-FOO@RIPE AS-BAR@RIPE"),
```

Comment:
We should add this case:

`("RIPE::AS15562:AS-SNIJDERS", "RIPE::AS15562:AS-SNIJDERS"),`

and 

`("RIPE::AS15562:AS-SNIJDERS:AS-TEST", "RIPE::AS15562:AS-SNIJDERS:AS-TEST"),`

and

`("RIPE::AS15562:AS-SNIJDERS:AS5", "RIPE::AS15562:AS-SNIJDERS:AS5"),`

Reply:
Bit late since it's already merged, you could do another PR tho :)